tests/test_observations.py

```diff
@@ -1,9 +1,21 @@
+# Required to Postponed evaluation of annotations (i.e. UnifiedGenerator),
+# allowing itto be imported in the TYPE_CHECKING block
+from __future__ import annotations
```

Comment:
See https://github.com/glass-dev/glass/pull/703, can we remove the comment here? Another thing to maintain